File: students/students/src/MON.Models/Ores/OresModel.cs
namespace MON.Models
{
    using System;
    using System.Collections.Generic;

    public class OresModel
    {
        public int? Id { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public int OresTypeId { get; set; }
        public string Description { get; set; }
        public IEnumerable<DocumentViewModel> Documents { get; set; }
        public int? PersonId { get; set; }
        public int? ClassId { get; set; }
        public int? InstitutionId { get; set; }
    }
}

File: students/students/src/MON.SPA/src/models/oresModel.js
import { DocumentModel } from '@/models/documentModel.js';
import Constants from '@/common/constants.js';

export class OresModel {
  constructor(obj = {}, moment = {}) {
    this.id = obj.id;
    this.oresTypeId = obj.oresTypeId;
    this.oresTypeName = obj.oresTypeName;
    this.description = obj.description;
    this.personId = obj.personId;
    this.classId = obj.classId;
    this.institutionId = obj.institutionId;
    this.startDate = (obj.startDate && moment) ? moment(obj.startDate).format(Constants.DATEPICKER_FORMAT) : '';
    this.endDate = (obj.endDate && moment) ? moment(obj.endDate).format(Constants.DATEPICKER_FORMAT) : '';
    this.documents = obj.documents ? obj.documents.map(el => new DocumentModel(el)) : [];
  }
}
